components/prism-wolfram.js

```diff
@@ -0,0 +1,24 @@
+	'context': {
+		pattern: /\w+`+\w*/,
+		alias: 'class-name'
+	},
+	'blank': {
+		pattern: /\b\w+\_{1,3}\b/,
```

Comment:
`\w` includes the `_` character. This means that this regex is equivalent to `\b\w+_\b`. Is this still correct or is this a bug?

Reply:
Good catch. So to be a pattern must have at least one trailing `_` but can have up to three `_`, so
maybe `/\b[^_]\w*\_{1,3}\b/`?

Reply:
> at least one trailing `_` but can have up to three `_`

In that case, you want `\b\w+_\b`.

Reply:
Ok I just add that.